lib/osf-components/addon/components/ancestry-display/component.ts

```diff
@@ -0,0 +1,74 @@
+import { tagName } from '@ember-decorators/component';
+import { alias } from '@ember-decorators/object/computed';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import { allSettled, task } from 'ember-concurrency';
+import I18N from 'ember-i18n/services/i18n';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import NodeModel from 'ember-osf-web/models/node';
+import defaultTo from 'ember-osf-web/utils/default-to';
+import styles from './styles';
+import template from './template';
+
+@tagName('')
+@layout(template, styles)
+export default class AncestryDisplay extends Component.extend({
+    getAncestors: task(function *(this: AncestryDisplay) {
+        if (!this.node) {
+            return [];
+        }
+
+        const parentId = this.node.belongsTo('parent').id();
+        const rootId = this.node.belongsTo('root').id();
+
+        // No ancestors
+        if (this.node.id === rootId) {
```

Comment:
How about `this.node.isRoot`? Actually, could be combined with the first `if` above
```ts
if (!this.node || this.node.isRoot) {
  return [];
}
```